packages/valory/skills/price_estimation_abci/rounds.py

```diff
@@ -150,7 +155,12 @@ class EstimateConsensusRound(CollectSameUntilThresholdRound):
 
 
 class TxHashRound(CollectSameUntilThresholdRound):
-    """A round in which agents compute the transaction hash"""
+    """
+    A round in which agents compute the transaction hash.
+
+    This is a special kind of round. It is a mix of collect same and collect different.
+    In essence, it collects the same values for the tx hash and different values for the signatures and the data.
+    """
```

Comment:
We might want to introduce this to the core at some point. It can be a useful feature; it helped us avoid two extra rounds compared to the implementation without this special round (#51).